test/ignore_SUITE.erl

```diff
@@ -76,6 +78,31 @@ hank_individual_rules(_Config) ->
 
     {comment, ""}.
 
+%% @doc No warning should be emmited for lists of ignored rules and neither should
+%%      evaluation of the code fail
+rebar_config_ignore(_Config) ->
+    Rule1 = unused_macros,
+    Rule2 = unnecessary_function_arguments,
+    FileErl = "rebar_config_ignore.erl",
+    Rules = [Rule1, Rule2],
+    IgnoreOld = [{FileErl, Rule1}, {FileErl, Rule2}],
+    IgnoreNew = [{FileErl, [Rule1, Rule2]}],
```

Comment:
I'd avoid _New/Old_ terminology, when new will be old next, and so on... maybe `IgnoreTuples`/`IgnoreOneTuple`

Reply:
Care to propose something so I can just accept via the UI?

Reply:
Does this mean you want me to change anything? Or is this a general comment?

Reply:
I avoided the issue by removing the variables: f36180c.